stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -2012,7 +2015,7 @@ class EnumTypeInfoBuilder {
     // with zero-sized payloads get treated for layout purposes as non-payload
     // cases.
     EnumKind Kind;
-    switch (GenericPayloadCases + NonGenericEmptyPayloadCases + NonGenericNonEmptyPayloadCases) {
+    switch (IndirectPayloadCases + GenericPayloadCases + NonGenericEmptyPayloadCases + NonGenericNonEmptyPayloadCases) {
```

Comment:
Further down, the existence of any "GenericPayloadCases" means we can't use spare bits but must rely entirely on an added tag.  This is why cases that are both generic and indirect now get counted as indirect.